sql/pgtap.sql.in

```diff
@@ -6375,7 +6375,7 @@ BEGIN
                     END IF;
                 END IF;
 
-            EXCEPTION WHEN raise_exception THEN
+            EXCEPTION WHEN OTHERS THEN
```

Comment:
This needs to be `OTHERS OR ASSERT_EXCEPTION` like the `lives_ok()/throws_ok()` handlers.